tests/l2/README.md

```diff
@@ -28,6 +28,10 @@ $ oc logs intel-sgx-job-4tnh5
 ```
 ### Verify Intel® Data Center GPU provisioning
 This workload runs [clinfo](https://github.com/Oblomov/clinfo) utilizing the i915 resource from GPU provisioning and displays the related GPU information.
+* Before deploying any workload, please ensure to set access for ```/dev/``` folder on the GPU node, with the command below:
+  
```

Comment:
Looks like we need a selinux policy for the workload pod? @mregmi? if it is the case, do we have an existing one can be used by the workload pod?

Reply:
We can use` container_device_t` selinux policy. Works with a custom ocp scc.
 I wonder how feasible it is to use for every workload/e2e with other operators/dependencies. @mregmi @uMartinXu thoughts?

Reply:
https://github.com/intel/intel-technology-enabling-for-openshift/pull/107 with `container_device_t` does not include enough permissions to run. Might require more permissions. For this release, I suggest we use this PR. Will keep the other PR open and verify with correct permissions. @mregmi @uMartinXu

Reply:
The container_device_t did not work on 4.12. We might require more permissions. For 1.0.1 I suggest we continue with this PR and will keep this PR open to change permissions https://github.com/intel/intel-technology-enabling-for-openshift/pull/109 for next release.

Reply:
Yea for now lets go with bool. we will have to give extra permissions later